src/pages/editor/InstitutionSupport.tsx

```diff
@@ -0,0 +1,81 @@
+import LinkIcon from '@mui/icons-material/Link';
+import { LoadingButton } from '@mui/lab';
+import { InputAdornment, TextField, Typography } from '@mui/material';
+import { useMutation } from '@tanstack/react-query';
+import { Field, FieldProps, Form, Formik, FormikProps } from 'formik';
+import { useTranslation } from 'react-i18next';
+import { useDispatch, useSelector } from 'react-redux';
+import { updateCustomerInstitution } from '../../api/customerInstitutionsApi';
+import { PageSpinner } from '../../components/PageSpinner';
+import { setCustomer } from '../../redux/customerReducer';
+import { setNotification } from '../../redux/notificationSlice';
+import { RootState } from '../../redux/store';
+import { CustomerInstitution } from '../../types/customerInstitution.types';
+import { dataTestId } from '../../utils/dataTestIds';
+
+export const InstitutionSupport = () => {
+  const { t } = useTranslation();
+  const dispatch = useDispatch();
+  const customer = useSelector((store: RootState) => store.customer);
+
+  const customerMutation = useMutation({
+    mutationFn: (values: CustomerInstitution) =>
+      updateCustomerInstitution({ ...values, serviceCenterUri: values.serviceCenterUri }),
+    onSuccess: (response) => {
+      dispatch(setCustomer(response.data));
+      dispatch(setNotification({ message: t('feedback.success.update_customer'), variant: 'success' }));
+    },
+    onError: () => dispatch(setNotification({ message: t('feedback.error.update_customer'), variant: 'error' })),
+  });
+
+  return (
+    <>
+      {!customer ? (
+        <PageSpinner />
+      ) : (
+        <Formik initialValues={customer} onSubmit={async (values) => customerMutation.mutate(values)}>
```

Comment:
overflødig async?
```suggestion
        <Formik initialValues={customer} onSubmit={(values) => customerMutation.mutate(values)}>
```

Reply:
Ah, har blitt litt forvekslinger her ser jeg, for nå blir heller ikke feltet satt som default under submitting heller.

Enten så må denne være `onSubmit={async (values) => customerMutation.mutateAsync(values)}`, og da kan du bruke `isSubmitting` for `disabled` og `loading`, eller så kan denne gjøres synkron, og du baserer deg på `customerMutation` direkte i stedet.

Jeg liker kanskje førsnevnte best:
```suggestion
        <Formik initialValues={customer} onSubmit={async (values) => customerMutation.mutateAync(values)}>
```